subsys/bluetooth/controller/ll_sw/ll_adv.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct ll_adv_set {
+	u8_t  chl_map:3;
+	u8_t  filter_policy:2;
+#if defined(CONFIG_BLUETOOTH_CONTROLLER_PRIVACY)
+	u8_t      rl_idx : 5;
+	u8_t      own_addr_type : 2;
+	u8_t      id_addr_type : 1;
+	u8_t      id_addr[BDADDR_SIZE];
```

Comment:
@Vudentz I do get your point, but I think this is not worth the added conditional compilation that it would require. I would appreciate @jhedberg's input on this because I don't have a strong view on this, but we're not ever going to serialize a Resolving List or an Advertising Set, and I doubt we'll ever print them cast to an u32_t or similar.